crates/bevy_camera_controller/src/free_cam.rs

```diff
@@ -1,37 +1,59 @@
-//! A freecam-style camera controller plugin.
-//! To use in your own application:
-//! - Copy the code for the [`CameraControllerPlugin`] and add the plugin to your App.
-//! - Attach the [`CameraController`] component to an entity with a [`Camera3d`].
+//! A camera controller that allows the user to move freely around the scene.
 //!
-//! Unlike other examples, which demonstrate an application, this demonstrates a plugin library.
+//! Free cams are helpful for exploring large scenes, level editors and for debugging.
+//! They are rarely useful as-is for gameplay,
+//! as they allow the user to move freely in all directions,
+//! which can be disorienting, and they can clip through objects and terrain.
+//!
+//! Fly cams are a kind of free cam, designed for fluid "flying" movement.
```

Comment:
Don't know where to put this, but it's worth mentioning that our free cam does not really support controllers ATM

Reply:
Yeah, I hope to fix that before release and I don't want to have to remember to update it 😅